### ISSUE TITLE
Create "Add governance" button on Open Collective

### ISSUE THREAD

**Issue by thelastjosh** [metagov/gateway#185]
A button, located directly within Open Collective's interface, that creates a copy of our current Slack voting policy for that collective.

Button logic:
1. Upon click, go to page, "Information". Here, the user can read about the policy and click "Install".
2. Detect if the user has admin privileges in that collective. If not, throw error.
3. Go to page, "Platform Selection". Here, the user should select be able to select the platform where governance happens. Currently, Slack will be the only option.
4. Insert Slack OAuth login flow.
5. Go to page, "Manage Policy". Here, the user should be able to review a list of the current channels in the Slack, select one such channel, and then activate the policy. The user should also be able to turn off the policy or erase it entirely on this page.

@shaunagm

**Comment by shaunagm** [comment]
I started writing out a guess at how we'd implement this with PolicyKit but this might be a good use case for a minimum viable driver.

**Comment by thelastjosh** [comment]
Agreed, this doesn't seem like it needs PK at all, since there's no need for flexibility or customizability in this button / use-case.
